src/manifolds/VectorBundle.jl

```diff
@@ -314,78 +107,52 @@ struct VectorBundle{TVS<:VectorSpaceType, TM<:Manifold} <: Manifold
     M::TM
     VS::VectorBundleFibers{TVS, TM}
 end
-
-base_manifold(B::VectorBundle) = base_manifold(B.M)
-
 function VectorBundle(VS::TVS, M::TM) where {TVS<:VectorSpaceType, TM<:Manifold}
     return VectorBundle{TVS, TM}(VS, M, VectorBundleFibers(VS, M))
 end
-
-function representation_size(B::VectorBundle)
-    len_manifold = prod(representation_size(B.M))
-    len_vs = prod(representation_size(B.VS))
-    return (len_manifold + len_vs,)
-end
-
-TangentBundleFibers{M} = VectorBundleFibers{TangentSpaceType,M}
-TangentBundleFibers(M::Manifold) = VectorBundleFibers(TangentSpace, M)
-
-CotangentBundleFibers{M} = VectorBundleFibers{CotangentSpaceType,M}
-CotangentBundleFibers(M::Manifold) = VectorBundleFibers(CotangentSpace, M)
-
 TangentBundle{M} = VectorBundle{TangentSpaceType,M}
 TangentBundle(M::Manifold) = VectorBundle(TangentSpace, M)
-
 CotangentBundle{M} = VectorBundle{CotangentSpaceType,M}
 CotangentBundle(M::Manifold) = VectorBundle(CotangentSpace, M)
 
 """
-    bundle_projection(B::VectorBundle, x::ProductRepr)
+    FVector(type::VectorSpaceType, data)
 
-Projection of point `x` from the bundle `M` to the base manifold.
-Returns the point on the base manifold `B.M` at which the vector part
-of `x` is attached.
+Decorator indicating that the vector `data` from a fiber of a vector bundle
+is of type `type`.
```

Comment:
This word order seems more correct:

Decorator indicating that the vector `data` is from a fiber of a vector bundle
of type `type`.